README.md

```diff
@@ -50,7 +50,7 @@ Other interesting links are:
 
 * Our [Apiary Documentation](http://telefonicaid.github.io/fiware-cygnus/api/latest) if you want to know how to use our API methods for Cygnus.
 * [cygnus-ngsi](doc/cygnus-ngsi/integration) **integration** examples .
-* [cygnus-ngsi](https://edu.fiware.org/mod/resource/view.php?id=1006) **introductory course** in FIWARE Academy.
+* [cygnus-ngsi](https://edu.fiware.org/mod/resource/view.php?id=1015) **introductory course** in FIWARE Academy.
```

Comment:
Is this really part of release closing of a king of "piggybacked bugfix" in this PR?